apport/crashdb.py

```diff
@@ -123,7 +123,7 @@ def init_duplicate_db(self, path):
         cur.execute("PRAGMA integrity_check")
         result = cur.fetchall()
         if result != [("ok",)]:
-            raise SystemError("Corrupt duplicate db:" + str(result))
+            raise SystemError(f"Corrupt duplicate db:{str(result)}")
```

Comment:
suggestion (not-blocking): get rid of superfluous `str()` call

The f-string machinery will call it for you :)

```suggestion
            raise SystemError(f"Corrupt duplicate db:{result}")
```